assets/dev/js/editor/document/save/behaviors/footer-saver.js

```diff
@@ -101,12 +101,23 @@ module.exports = class FooterSaver extends Marionette.Behavior {
 
 	addTooltip() {
 		// Create tooltip on controls
-		this.$el.find( '.tooltip-target' ).tipsy( {
-			// `n` for down, `s` for up
-			gravity: 's',
-			title() {
-				return this.getAttribute( 'data-tooltip' );
-			},
+		this.$el.find( '.tooltip-target' ).each( ( index, button ) => {
+			const $button = jQuery( button );
+			let offset = 0;
+
+			if ( $button.attr( 'id' ) === this.ui.buttonSaveOptions.attr( 'id' ) ) {
```

Comment:
Make it generic. Use an attribute `tooltip-offset` to set a custom offset. then use `offset: $button.attr( 'tooltip-offset )`,

Reply:
Done